webrtc/RTCSctpTransport-constructor.html

```diff
@@ -0,0 +1,95 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>RTCSctpTransport constructor</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  'use strict';
+
+  // Test is based on the following editor draft:
+  // https://w3c.github.io/webrtc-pc/archives/20170515/webrtc.html
+
+  // Helper function to generate answer based on given offer using a freshly
+  // created RTCPeerConnection object
+  function generateAnswer(offer) {
+      const pc = new RTCPeerConnection();
+      return pc.setRemoteDescription(offer)
+      .then(() => pc.createAnswer());
+  }
```

Comment:
Removed the helper code now that I can directly import it from the merged helper.js